api/src/main/java/org/pmiops/workbench/config/WebMvcConfig.java

```diff
@@ -66,56 +60,6 @@ public DbUser user(UserAuthentication userAuthentication) {
     return userAuthentication.getUser();
   }
 
-  @Bean
-  public WorkbenchEnvironment workbenchEnvironment() {
-    return new WorkbenchEnvironment();
-  }
-
-  /**
-   * Service account credentials for Gsuite administration. These are derived from a key JSON file
```

Comment:
The change to this method (now located in StoredCredentialsConfig) should be the only functional change in this PR. Instead of loading the gsuite-admin SA creds from the App Engine resource package, I decided to follow-up on my prior TODO and switch to loading them from GCS. What this effectively means is that each App Engine task will talk to GCS when the first API call is made that requires G Suite interactions (e.g. creating a user). This bean is singleton-scoped, so the credentials should be loaded and cached for the lifetime of each GAE task.

Reply:
But it still needs to be refreshed during the task lifetime?